### PR TITLE
Lock body/container height on iOS7 upwards

### PR DESCRIPTION
**Pull request by donnielrt**
Status: **Opened for visibility**
Reviewers: @scalvert @kpeatt @ericmuyser @Helen-Mobify 

## Related PRs
https://github.com/mobify/pinny/pull/84

## TODOs
- [x] Test on all target devices
- [x] Engineers +2

Use iOS7 background scrolling fix with lockup. Also fixes an issue with input focus competing with browser scroll on focusing on inputs.

## Changes
- Set container/body heights on iOS7+. Earlier we had a special case for iOS7, which didn't lock background scrolling.

### CODE REVIEW

File: dist/lockup.js [hunk 1]
```diff
@@ -156,27 +154,9 @@
                     .height(window.innerHeight)
                     .css('overflow', 'hidden')
                     .scrollTop(this.scrollPosition - getPadding('top') - getPadding('bottom'));
-
```
Comment (donnielrt) [inline dist/lockup.js]: @kpeatt I think you might've had some concerns about this, let's chat when you have a moment

**Comment by scalvert** [comment]
This seems like an OK fix to me, but again this will mean that we're going with locking up always, but adding a flash on repaint. Have we agreed this is the way we want to go moving forward?

**Comment by donnielrt** [comment]
@scalvert couldn't see a flash on repaint on iPad Mini 7.1, but I can see this possibly being an issue on other projects, maybe?

**Comment by kpeatt** [comment]
@donnielrt Definitely causes repaint issues on both iPads and iPhones. I can show it to you when I'm in office next.

**Comment by donnielrt** [comment]
@kpeatt kk, sounds good. Gonna leave it in BTR for the time-being till we figure this out.

**Comment by Helen-Mobify** [comment]
@donnielrt I see repaint flash as well

**Comment by donnielrt** [comment]
@Helen-Mobify @fractaltheory can we sit down quickly for a bit next week to nail down and resolve this issue?

**Comment by Helen-Mobify** [comment]
sure

**Comment by fractaltheory** [comment]
sounds good

**Comment by kpeatt** [comment]
:-1:

**Comment by donnielrt** [comment]
@kpeatt too late, I'm already growing a beard to take your place

**Comment by donnielrt** [comment]
Hey, we’re looking to prune older abandoned PRs. If this PR is still relevant and you would like to see it merged in, please reopen the PR and we’ll add it to our backlog! Thanks!

**Comment by ericmuyser** [comment]
+1 Code looks good and is working on BTR

**Comment by donnielrt** [comment]
@ericmuyser is this branch in use on BTR? Do we need to reopen this branch?

**Comment by donnielrt** [comment]
Reopening along with https://github.com/mobify/pinny/pull/84.

File: dist/lockup.js (hunk 1 above)
Comment (donnielrt) [inline dist/lockup.js]: @kpeatt still waiting!

**Pull request merged**